common/rya.api/src/main/java/mvm/rya/api/persist/index/RyaSecondaryIndexer.java

```diff
@@ -34,7 +34,7 @@
 
 public interface RyaSecondaryIndexer extends Closeable, Flushable, Configurable {
 
-    public String getTableName();
+    public String getTableName(); // TODO: some indexers have multiple tables, should return a list.
```

Comment:
change to block comment, for now those indexers can return a comma delimited string?